ocrd/ocrd/cli/workspace.py

```diff
@@ -288,10 +309,16 @@ def workspace_cli_bulk_add(ctx, regex, mimetype, page_id, file_id, url, file_grp
         if not m:
             if skip:
                 continue
-            log.error("File not matched by regex: '%s'" % file_path)
+            log.error("File '%s' not matched by regex: '%s'" % (file_path, regex))
             sys.exit(1)
         group_dict = m.groupdict()
 
+        # derive --file-id from filename if not --file-id not explicitly set
+        if not file_id:
+            file_id = safe_filename(str(file_path))
+
+        # if not url
```

Comment:
Is this a placeholder for a TODO?

Reply:
good catch, looks like an artefact of refactoring, should be removed.